app/components/sign-up/signup-form.js

```diff
@@ -0,0 +1,21 @@
+import Component from '@glimmer/component';
+import { action } from '@ember/object';
+
+export default class SignupComponent extends Component {
+  get label() {
+    const { state } = this.args;
+
+    const labelText = {
+      firstName: 'What is your First Name?',
+      lastName: 'And what\'s is your Last Name?',
+      userName: 'Now choose your awesome username!',
+    };
+
+    return labelText[state];
+  }
+
+  @action inputFieldChanged({ target: { value } }) {
```

Comment:
I see that where it's being reference, no value is being passed in the component :o 
But since this works, I might not know this way :(
Can you reference the link so I can read up and learn please?

Reply:
So this function receives the `event` object as the argument. This function is registered with the `on input` event.
So, have destructured the `event` object here.

Reply:
Oh yeah, that makes sense